stellarphot/settings/models.py

```diff
@@ -80,3 +89,107 @@ class PhotometryFileSettings(BaseModel):
     aperture_locations_file: Path = Field(
         filter_pattern=["*.ecsv", "*.csv"], default=""
     )
+
+
+class TimeType(Time):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
+        return Time(v)
+
+
+class SkyCoordType(SkyCoord):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
+        return SkyCoord(v)
+
+
+class Exoplanet(BaseModel):
+    """
+    Create an object representing an Exoplanet.
+
+    Parameters
+    ----------
+
+    epoch : TimeType
+        Epoch of the exoplanet as an astropy Time object, not required.
+
+    period : QuantityType
+        Period of the exoplanet as a Quantity with units of time, not required.
+
+    Identifier : Str
+        Identifier of the exoplanet as a string, required.
+
+    coordinate : SkyCoordType
+        Coordinates of the exoplanet as an astropy SkyCoord object, required.
+
+    depth : float
+        Depth of the exoplanet object as a float, not required.
+
+    duration : QuantityType
+        Duration of the exoplanet transit as a Quantity with units of time,
+        not required.
+
+    Examples
+    --------
+
+    To create an `Exoplanet` object, you can pass in the epoch,
+    period, Identifier, coordinate, depth, and duration as keyword arguments:
+
+    >>> planet  = Exoplanet(epoch=Time(0, format="jd"), period=0 * u.min,
```

Comment:
It is probably more helpful if you put in information for a real exoplanet. Maybe Kelt-1?

Reply:
updated to use kelt-1b info but all of it may not be correct. kinda estimated values based on the graphs from a paper i found. also the depth is based on normalized flux

Reply:
You can look up exoplanet data here: https://exoplanetarchive.ipac.caltech.edu/

There are likely several values for kelt-1, maybe use the most recent.